.github/workflows/source-package.yml

```
@@ -0,0 +1,30 @@
+name: Build a source package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+        # Don't use checkout, we need a full clone for this workflow
+      - name: Clone repository
+        run: |
+          git clone https://github.com/yubico/yubioath-desktop
+
+      - name: Build a .tar.gz source package
+        run: |
+          export REF=$(echo ${GITHUB_REF} | cut -d '/' -f 3)
```

Same thing here :slightly_smiling_face:

```suggestion
          export REF=${GITHUB_REF#refs/*/}
```